features/admin.extensions.v1/configs/application.tsx

```diff
@@ -149,6 +149,11 @@ export const applicationConfig: ApplicationConfig = {
             ApplicationManagementConstants.DEVICE_GRANT,
             ApplicationManagementConstants.ORGANIZATION_SWITCH_GRANT,
             ApplicationManagementConstants.OAUTH2_TOKEN_EXCHANGE
+        ],
+        [ "sub-organization-application" ]: [
```

Comment:
If this is common for both onprem and SaaS, let's define this in applications.v1 module itself. extensions.v1 is meant for configs that can be changed between onprem and SaaS products.

Reply:
This will be converted into a template and this is the common way to handle that. So IMO we can keep this.